tests/test_user_remove_association.py

```diff
@@ -0,0 +1,87 @@
+# Installed
+import json
+import http
+import copy
+
+# Own
+import tests
+from tests.test_project_creation import proj_data_with_existing_users
+
+
```

Comment:
Add test for if the project doesn't exist maybe?

Reply:
Done.